File: Julia/is_prime.py
from math import sqrt
def is_prime(n):
	if n<= 3:
		return True
	if n%2 == 0:
		return False
	for i in range(3,sqrt(n),2):
		if n%i==0:
			return False
	return True

print(is_prime(100))

File: Julia/is_prime.jl
function is_divisible(n::Int64, i::Int64)
    return n%i == 0
end


function is_prime(n::Int64)
    if n <= 3
        return true
    end
    if is_divisible(n,2)
        return false
    end
    i = 3
    while i <= sqrt(n)
        if is_divisible(n,i)
            return false
        end
        i+=2
    end

    return true

end


println(is_prime(100))
